api/tests/unit/organisations/test_unit_organisations_models.py

```diff
@@ -193,19 +182,14 @@ def test_organisation_is_auto_seat_upgrade_available(organisation, settings):
     assert organisation.is_auto_seat_upgrade_available() is True
 
 
-class SubscriptionTestCase(TestCase):
-    def setUp(self) -> None:
-        self.organisation = Organisation.objects.create(name="Test org")
-
-    def tearDown(self) -> None:
-        Subscription.objects.all().delete()
-
-    def test_max_seats_set_as_one_if_subscription_has_no_subscription_id(self):
-        # Given
-        subscription = Subscription.objects.get(organisation=self.organisation)
+def test_organisation_max_seats_set_as_one_if_subscription_has_no_subscription_id(
```

Comment:
nit: the test name is slightly confusing, can we change it to something like: `test_organisation_default_subscription_have_one_max_seat`

Reply:
Sure that's easy